client/config.go

```diff
@@ -0,0 +1,61 @@
+package client
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+// Config contains the client's configuration needed to send request to a
+// CONIKS-server:
+//  - the path to the server's VRF and signing public-key file, respectively;
+//  - the actual public-keys parsed from these files.
+type Config struct {
+	SignPubkeyPath string `toml:"sign_pubkey_path"`
+	VrfPubkeyPath  string `toml:"vrf_pubkey_path"`
+
+	SigningPubKey sign.PublicKey
+	VrfPubKey     vrf.PublicKey
```

Comment:
It should be used or not? (IIRC it was used before I changed the code to using the consistency-checks). Hope I'll find some time to look into this issue later.